bpmn-intellij-plugin-core/src/main/kotlin/com/valb3r/bpmn/intellij/plugin/core/render/IconProvider.kt

```diff
@@ -105,6 +106,7 @@ data class IconProviderImpl(
         override val gridDense: Icon = IconLoader.getIcon("/icons/actions/grid.png"),
         override val noGrid: Icon = IconLoader.getIcon("/icons/actions/no-grid.png"),
         override val gear: Icon = IconLoader.getIcon("/icons/ui-icons/gear.png"),
+        override val triggered: Icon = IconLoader.getIcon("/icons/ui-icons/triggired.png"),
```

Comment:
Icon should support both dark and light editor themes (currently it looks bad in Darcula theme)